### DISCUSSION TITLE
working on a new cross-platform network relay that is a full virtualized network

### DISCUSSION THREAD

**Discussion by progrium** [copy/v86#1064] [Show and tell]
![v86_multiplayer](https://github.com/copy/v86/assets/647/3f3c58e4-bb5b-4326-85fd-0dc1c7ffc399)

**Comment by copy** [comment]
Looking forward to it!

**Comment by bit-lang** [comment]
how's it going?

**Reply by SuperMaxusa** [reply]
Here is repo: https://github.com/progrium/go-netstack, the author is working on new projects with this networking stack: https://github.com/copy/v86/discussions/1085

**Reply by bit-lang** [reply]
thank you. you guys are my heroes
